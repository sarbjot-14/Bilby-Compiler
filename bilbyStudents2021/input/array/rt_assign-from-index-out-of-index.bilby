main{
	imm integers := [11,12,13,14,15];
	
	imm a := integers[-1];
	
	
	%imm b := integers[5];
	
	

}