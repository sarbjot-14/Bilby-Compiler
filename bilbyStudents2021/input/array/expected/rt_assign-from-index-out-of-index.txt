Runtime error: indexing out of bounds array
